definitions/npm/rxjs_v5.0.x/flow_v0.34.x-/rxjs_v5.0.x.js

```diff
@@ -1657,3 +1657,276 @@ declare module "rxjs/util/UnsubscriptionError" {
     UnsubscriptionError: typeof rxjs$UnsubscriptionError,
   };
 }
+
+declare module "rxjs/observable/create" {
+    declare module.exports: {
+        create: typeof rxjs$Observable.create,
+    };
+}
+
+declare module "rxjs/observable/bindCallback" {
+    declare module.exports: {
+        bindCallback: typeof rxjs$Observable.bindCallback,
+    };
+}
+
+declare module "rxjs/observable/bindNodeCallback" {
+    declare module.exports: {
+        bindNodeCallback: typeof rxjs$Observable.bindNodeCallback,
+    };
+}
+
+declare module "rxjs/observable/combineLatest" {
+    declare module.exports: {
+        combineLatest: typeof rxjs$Observable.combineLatest,
+    };
+}
+
+declare module "rxjs/observable/concat" {
+    declare module.exports: {
+        concat: typeof rxjs$Observable.concat,
+    };
+}
+
+declare module "rxjs/observable/defer" {
+    declare module.exports: {
+        defer: typeof rxjs$Observable.defer,
+    };
+}
+
+declare module "rxjs/observable/empty" {
+    declare module.exports: {
+        empty: typeof rxjs$Observable.empty,
+    };
+}
+
+declare module "rxjs/observable/forkJoin" {
```

Comment:
Write a test or two to smoke screen that this works. Bonus points for `describe/it` blocks.